index.js

```diff
@@ -95,7 +104,8 @@ export default class LinterJSCS {
         this.jscs.registerDefaultRules();
 
         const filePath = editor.getPath();
-        const config = configFile.load(false, path.dirname(filePath));
+        const config = configFile.load(false,
+          path.join(path.dirname(filePath), this.configPath));
```

Comment:
You could also just use [this](https://github.com/AtomLinter/atom-linter/blob/master/lib/helpers.coffee#L121), which would search the current directory and any parents.